conda_forge_tick/migrators.xsh

```diff
@@ -301,10 +301,14 @@ class Compiler(Migrator):
 
     def migrate(self, recipe_dir, attrs, **kwargs):
         self.out = $(conda-smithy update-cb3 --recipe_directory @(recipe_dir))
+        return True
 
     def pr_body(self):
         body = super().pr_body()
-        body.format('{}\n'
+        body = body.format('{}\n'
```

Comment:
`.format(self.out)` should come here

Reply:
Will it properly do the string formatting?

Reply:
Ignore my comment. It should work fine